### PR TITLE
Fix: Renamed duplicate "Artist" to "Contributing Artists" in properties window

### LINKED ISSUE #13423: Bug: Rename duplicate "Artist" to "Contributing Artists" in properties window
### Description

image
The "Album Artist" can only show the last person in that section, and the first "Artist"(which doesn't exist in other apps) seems to be copying "Album Artist".

### Steps To Reproduce

Open the Properties of any audio file, switch to Details, and that should happen.

### Requirements

- Rename duplicate "Artist" to "Contributing Artists" in properties window (match File Explorer)

### Files Version

2.5.21.0

### Windows Version

10.0.22621.2283

### Log File

[debug.log](https://github.com/files-community/Files/files/12708229/debug.log)

### PR DESCRIPTION
**Pull request by hemalshah28**
**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #13423

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots (optional)**
Add screenshots here.

Before fix
image

Just for testing I have added one more artist in the same field.

After fix
image

Preview pane fix
image

### CODE REVIEW

File: src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256 [hunk 1]
```diff
@@ -1 +1 @@
-9f09add6084f2b30451d91864a573cc4cd0bf728a1df9bfc87fd1b7027487c60
+881d6caef54765251a7e9c88df9a090c5dd939b17140c199d3d08259d5ced961
```
Comment (yaira2) [inline src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256:1]: Can you revert this change?

Comment (hemalshah28) [reply src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256]: Done. Reverted changes.

File: src/Files.App/Resources/PropertiesInformation.json [hunk 2]
```diff
@@ -205,7 +205,7 @@
 		"ID": null
 	},
 	{
-		"NameResource": "Artist",
+		"NameResource": "Contributing Artist",
```
Comment (0x5bfa) [inline src/Files.App/Resources/PropertiesInformation.json:208]: Do not insert a space.

File: src/Files.App/Strings/en-US/Resources.resw [hunk 3]
```diff
@@ -3529,4 +3529,7 @@
   <data name="CannotInitializeGitRepo" xml:space="preserve">
     <value>Files can't initialize this directory as a Git repository.</value>
   </data>
+  <data name="Contributing Artist" xml:space="preserve">
```
Comment (0x5bfa) [inline src/Files.App/Strings/en-US/Resources.resw:3532]: As above

**Review by 0x5bfa — COMMENTED** [review]
Thank you for opening your first time PR.

File: src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256 (hunk 1 above)
Comment (hishitetsu) [reply src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256]: The update to Files.App.Launcher.exe should also be reverted.

File: src/Files.App/Resources/PropertiesInformation.json (hunk 2 above)
Comment (hemalshah28) [reply src/Files.App/Resources/PropertiesInformation.json:208]: Done. Thanks for your review.

File: src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256 (hunk 1 above)
Comment (hemalshah28) [reply src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256]: Done. Thanks for review

File: src/Files.App/Strings/en-US/Resources.resw (hunk 3 above)
Comment (hemalshah28) [reply src/Files.App/Strings/en-US/Resources.resw]: Done.

**Review by hishitetsu — COMMENTED** [review]
Wouldn't the same modification be needed for PreviewPanePropertiesInformation.json?

**Comment by hemalshah28** [comment]
> Wouldn't the same modification be needed for PreviewPanePropertiesInformation.json?

yes you are right. It's good catch.

**Review by 0x5bfa — APPROVED** [review]
LGTM

**Review by hishitetsu — APPROVED** [review]
LGTM. Thanks for your contribution.
